docs/howto/configure-global-pull-secrets.md

```diff
@@ -0,0 +1,53 @@
+---
+tags:
+  - alpha
+---
+
+# Configure global pull secrets for allowing components to pull private images
```

Comment:
```suggestion
# Configure global pull secrets for allowing components to pull from private registries
```

I think it would be more clearer to use the term "private registries" instead of "private images," as the authentication is specifically for accessing the registry where images are hosted, right?